_posts/2018-07-25-the_incredibles_3.md

```diff
@@ -0,0 +1,254 @@
+they're all kids, I feel like this shouldn't be, like, national or international
+conspiracy stuff, but maybe that would be OK. I don't want to blow the top off
+the setting, though.
+
+When they surface this revelation, though, Peter doesn't assimilate it well. He
+has a pretty absolutist moral model where if you've broken the law, you're a Bad
+Guy™ and have to be stopped and punished. He also, you know, doesn't value
+Khaled (around here they unmask him) as much as his other peers at school, so
+it's easy for him to over-punish him and still feel good about himself. Peter
+tries to manipulate Violet into seeing things his way and maybe beating the hell
+out of Khaled.
+
+Violet, for the first time, sees what Peter is doing clearly and rejects it. She
+ends the encounter, but nothing is resolved. Khaled is in the wind and she and
+Peter are clearly done, but it isn't clear what the next thing is. She feels
+lost. Ideally, too, there's some school-related parallel, here. Some deadline or
+big event that I just haven't worked out (yet?). In the best tradition of _Buffy
+the Vampire Slayer_ and _Veronica Mars_, the "hero" plot should be paralleld by
+a "normal adolescent" plot.
+
+Feeling uncertain and not sure where to turn, she goes to visit Edna. Mostly
+this is just because I love Edna as a character and I want her to appear at
+least a couple times in this plot, but also because I feel like teenagers tend
+to listen to advice better from adults who are not their parents.
+
+Anyway, Edna gives Violet advice and validates her creeped-out feelings about
+Peter. She also advises Violet on verifying the merit of Khaled's claim about a
+conspiracy. Violet does some self-actualization and as a visual parallel, Edna
+gives her a new costume (and, it is revealed, had already been designing it for
+her, "Because a woman must stand on her own."). The new costume isn't the
+red/black/yellow Incredibles theme, but is it's own thing. It should reference
+the family design in a small way so symbolize her being her own person, but not
+entirely separated from her family.
+
+So, in her new costume, Violet goes out and finds Khaled. Khaled shows her his
+evidence so far and they team up to get the last bit of evidence they need to
+prove and defang the conspiracy. When they get there, of course, Peter also
+arrives and they have it out verbally and then physically.
+
+In here there's a crucial moment where Khaled suggests to Violet that she do
+something she doesn't feel comfortable with; something that Peter had previously
+gotten her to do. She declines and he responds with something like, "Oh. Of
+course. You're right," and they figure out another way to achieve their goals.
+Eventually they overcome Peter, despite his strength, get the evidence and, you
+know, win the day or whatever. Importantly, Violet and Khaled do not begin a
+romantic relationship; just a friendship and a partnership.
```

Comment:
OK prior to this last line I was going to ask "isn't she just substituting one flavor of heteronormative coupling for another?"  Nice.